### DISCUSSION TITLE
Empty data rulesets in a data grid

### DISCUSSION THREAD

**Discussion by rahulpd101** [ng-matero/ng-matero#608] [Q&A]
I am using mtx-grid to make a table where I am receiving an object with a key whose value is "" (an empty string). This empty string is shown by the table as '-' (a small dash).
I want to show a mat-icon in place of that small dash. How can I do so?
I have tried using ::before pseudo-selector to show content but they appear as very small icons. Hence, I want to use mat-icons instead.

**Comment by nzbin** [comment]
You can use the prop `emptyValuePlaceholder` to customize, but it only support `string` type.
